subscription-manager.spec

```diff
@@ -116,6 +116,13 @@ from the server. Populates /etc/docker/certs.d appropriately.
 %{_sysconfdir}/rhsm/pluginconf.d/container_content.ContainerContentPlugin.conf
 %{rhsm_plugins_dir}/container_content.py*
 %{_datadir}/rhsm/subscription_manager/plugin/container.py*
+# Copying Red Hat CA cert into each directory:
+%dir %{_sysconfdir}/docker/certs.d/cdn.redhat.com
+%dir %{_sysconfdir}/docker/certs.d/access.redhat.com
+%dir %{_sysconfdir}/docker/certs.d/registry.access.redhat.com
+%{_sysconfdir}/docker/certs.d/cdn.redhat.com
+%{_sysconfdir}/docker/certs.d/access.redhat.com
+%{_sysconfdir}/docker/certs.d/registry.access.redhat.com
```

Comment:
For my test install, with selinux-policy-3.13.1-17.el7.noarch

I see different default context/labels for

```
/etc/rhsm/ca/redhat-uep.prm
[subscription-manager (dgoodwin/docker-config % u=)]$ ls -laZ /etc/rhsm/ca/redhat-uep.pem 
-rw-r--r--. root root system_u:object_r:etc_t:s0       /etc/rhsm/ca/redhat-uep.pem
[subscription-manager (dgoodwin/docker-config % u=)]$ matchpathcon /etc/rhsm/ca/redhat-uep.pem 
/etc/rhsm/ca/redhat-uep.pem	system_u:object_r:etc_t:s0
```

```
[subscription-manager (dgoodwin/docker-config % u=)]$ ls -laZ /etc/docker/certs.d/access.redhat.com/redhat-uep.crt 
-rw-r--r--. root root system_u:object_r:cert_t:s0      /etc/docker/certs.d/access.redhat.com/redhat-uep.crt
[subscription-manager (dgoodwin/docker-config % u=)]$ matchpathcon /etc/docker/certs.d/access.redhat.com/redhat-uep.crt
/etc/docker/certs.d/access.redhat.com/redhat-uep.crt	system_u:object_r:cert_t:s0
```